main.js

```diff
@@ -93,7 +104,9 @@ const askForSecrets = async () => {
 	if (config.get("discordBot")) {
 		dc = new discord.Client();
 		dc.login(discordBotToken).catch(()=>{
-			console.warn("There was an error when trying to log in using the provided Discord bot token."); //handle empty tokens gracefully
+			if (config.Bot != "1"){
```

Comment:
Change this check to 
```suggestion
			if (config.discordBot){
```